draft-issuance.md

```diff
@@ -0,0 +1,175 @@
+
+Instead of following a step function around the four-year halving cycle inherited
+from Bitcoin, we propose a slow exponential “smoothing” of the curve. The new issuance
+scheme would approximate the current 4 year cycle, and results in the last
+zatoshi being issued in around 114 years.
+
+# Motivation
+
+Zcash’s economic model is inherited from Bitcoin and includes the concept of a halving
+mechanism to regulate the issuance of new coins. This approach, though foundational, invites
+ a reevaluation amid Zcash’s ongoing evolution. As the network matures, the need to address
+potential future challenges and ensure a sustained and stable economic ecosystem becomes
+apparent. The transition to a smoothed emissions curve offers an opportunity to adjust the network's
+issuance dynamics while maintaining the supply cap of 21,000,000 coins. By doing so, Zcash
+endeavors to optimize its economic framework, accommodating changing circumstances while
+maintaining predictability and stability in rewards distribution.
+
+This proposal outlines a solution to address challenges associated with the existing block
+subsidy issuance mechanism in the Zcash network. The primary goal of this proposal is to
+introduce a more predictable and stable issuance of ZEC by smoothing out the issuance
+curve while preserving the supply cap. It's important to note that this proposal does
+not seek to alter the fundamental aspects of Zcash's issuance policy. The average block
+subsidy amount over time will remain the same and the funds for block subsidies will last
+a similar amount of time. Instead, it focuses solely on enhancing the predictability
+and consistency of the block subsidy issuance process.
+
+Smoothing the emissions curve helps ensure that the network remains economically
+viable and stable as it transitions from a traditional issuance mechanism to one
+that maintains a sustainable and predictable issuance of rewards over time. It
+prevents abrupt changes in the rate of newly issued coins, which could lead to
+disruptions in the network's economic model and potentially impact its security
+and sustainability. A smoother emissions curve allows for a more gradual and controlled
+transition, providing ZEC stakeholders and participants with a clear understanding of
+how rewards will be distributed over time.
+
+
+
+# Specification
+
+Smoothing the issuance curve is possible using an exponential decay formula that
+satisfies the following requirements:
+
+## Issuance Goals
+
+1. Block subsidies are monotonically decreasing, as long as `ZsfBalanceAfter(h)` is monotonically decreasing
+2. Block subsidies approximate a continuous function
+3. When `ZSF_BALANCE(h) > 0` then block subsidies for block `h`
+MUST always be `> 0`, preventing a final “unmined” zatoshi
+4. For any 4 year period, all paid out block subsidies MUST equal approximately
+half of `ZSF_BALANCE` at the beginning of that 4 year period
```

Comment:
We think it is possible to simplify these requirements, because they are not a strict specification.

The old requirement 1 is redundant, because it is implied by the old requirements 2 and 4.

(Apologies for my original suggestion here, the ZIP Editors have decided that these are best described as requirements. See ZIP-244 for an example.)

```suggestion
# Requirements

Smoothing the issuance curve is possible using an exponential decay formula that
satisfies the following requirements:

## Issuance Requirements

1. The issuance can be summarised into a reasonably simple explanation
2. Block subsidies approximate a continuous function
3. If there are funds in the ZSF, then the block subsidy must be non-zero, preventing any final “unmined” zatoshis
4. For any 4 year period, all paid out block subsidies are approximately equal to half of the ZSF at the beginning of that 4 year period, if there are no deposits into the ZSF during those 4 years
TODO daira: add a requirement that makes the initial total issuance match the previous total issuance  
```